README.md

```diff
@@ -1,3 +1,112 @@
 # adage-server
 
 [![Code Climate](https://codeclimate.com/github/greenelab/adage-server/badges/gpa.svg)](https://codeclimate.com/github/greenelab/adage-server)
+
+## Get a working instance of the adage-server running:
+
+### Clone the adage-server repository
+```shell
+git clone git@github.com:greenelab/adage-server.git
+```
+
+### Edit settings in config.py file
+Set the full path of parent directory where adage-server repository was
+cloned into in the `'home_dir'` key of the `OS_CONFIG` setting.
+```python
+OS_CONFIG = {
+    'home_dir':  '/<path_to_directory>/',
+}
+
+```
+
+Set up Database Name, User, Password and Host in `databases` settings:
+```python
+CONFIG.update({
+    'databases': {'default': {
+                     'ENGINE': 'django.db.backends.postgresql_psycopg2',
+
+                      # database name and user cannot have upper case letters
+                      'NAME': '<your_adage_database_name>',
+                      'USER': '<your_adage_db_username>',
+                      'PASSWORD': '<your_db_username_password>',
+
+                      # Wherever PostgreSQL is being hosted,
+                      # usually localhost for development
+                      'HOST': 'localhost',
+
+                      # Port where it is being hosted from,
+                      # usually 5432
+                      'PORT': '5432',
+                     }
+                 },
+
+})
+```
+
+### Set up Database Name, User, Password and Host on PostgreSQL
```

Comment:
Maybe preface this with: example setup using postgresql. Don't think we want to consider postgresql required unless we know other databases won't work.